app/models/competition.rb

```diff
@@ -2847,7 +2858,8 @@ def self.form_data_json_schema
           "type" => "object",
           "properties" => {
             "isConfirmed" => { "type" => "boolean" },
-            "isVisible" => { "type" => "boolean" },
+            "autoAcceptEnabled" => { "type" => %w[boolean null] },
+            "autoAcceptDisableThreshold" => { "type" => %w[integer null] }, "isVisible" => { "type" => "boolean" }
```

Comment:
I think you're missing a line break here. On that note: Why did you add the autoAccept fields "sandwiched" between `isConfirmed` and `isVisible`, instead of just appending them behind `isVisible`?

Reply:
Oh, this is actually a duplicate now that you're exposing this under `competitorLimit` as well

Reply:
Resolved